.github/workflows/reusable-build.yml

```diff
@@ -123,6 +123,7 @@ jobs:
       run: |
         choco install -y llvm --version 11.0.1 --allow-downgrade
         echo "C:\Program Files\LLVM\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+        where clang.exe
```

Comment:
nit: does this also help fail the CICD step if the installation of the tool fails?

Reply:
Yes, the exit code of the last command determines if the task succeeds or fails. The issue is that the choco install fails, but the next line (the "echo") succeeds, so the task wasn't being failed. By adding an explicit check for the binary, this will fail if the exe isn't present.

Reply:
Please add that fact to the PR description in github.  Thanks!